Vagrantfile

```diff
@@ -73,13 +87,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         mkdir -p /etc/salt
         echo \"event_name: ${EVENT_NAME}\" >> /etc/salt/grains
         echo \"event_year: ${EVENT_YEAR}\" >> /etc/salt/grains
+        echo \"is_vagrant_windows: ${IS_VAGRANT_WINDOWS}\" >> /etc/salt/grains
 "
 
     config.vm.provision :salt do |salt|
         salt.install_master = true
         salt.install_type = 'git'
         salt.install_args = 'v2018.3.2'
-        salt.version = '2018.3.2'
         salt.seed_master = {reggie: 'vagrant/vagrant/files/reggie.pub'}
```

Comment:
users were reporting issues with this line, removing it (it still specifies above the revision to use)